examples/test_fixtures/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_version = "~> 0.10.6"
+}
+
+provider "aws" {
+  region  = "${var.aws_region}"
+  version = "1.0.0"
```

Comment:
Don't specify version here when aws provider v1.0.1 is out because your code _should_ work with latest stable version of the provider.

Reply:
I would like to have versions in here to constrain which versions I know to be testing out. I've changed this to an optimistic version constraint which will use latest but nothing lower than specified.